micrometer-observation/src/main/java/io/micrometer/observation/Observation.java

```diff
@@ -1029,8 +1028,7 @@ public <T> Context put(Object key, T object) {
          * @return entry or {@code null} if not present
          */
         @Override
-        @Nullable
-        public <T> T get(Object key) {
+        @Nullable public <T> T get(Object key) {
```

Comment:
The annotation is in the wrong place (although effectively it will work anyway). This is because I ignored the micrometer-observation module when running the recipe to move the annotations, but the formatter ran here anyway. I think we can leave it like this for now and fix it in a separate change when we do the micrometer-observation module.